run/logging-manual/main.go

```diff
@@ -79,6 +79,9 @@ func (e Entry) String() string {
 // [START run_manual_logging]
 
 func indexHandler(w http.ResponseWriter, r *http.Request) {
+        // Disable log timestamps, these are added automatically.
+        log.SetFlags(0)
```

Comment:
Would expect this to be in `main`. But, do we need it here to make it work as a Cloud Function? Perhaps an `init`?

Reply:
I agree. The reason I did it this way was to avoid adding the overhead of another snippet. I could wrap this in an `init` just above the handler to keep it grouped together.

Reply:
I think `init` makes more sense than setting it inside the handler? Neither is perfect.